content/docs/use-cases/sharing-data-and-model-files.md

```diff
@@ -1,35 +1,31 @@
 # Sharing Data and Model Files
 
-Like Git, DVC allows for a distributed environment and collaboration. We make it
-easy to consistently get all your data files and directories into any machine,
-along with matching source code. All you need to do is to setup
-[remote storage](/doc/command-reference/remote) for your <abbr>DVC
-project</abbr>, and push the data there, so others can reach it. Currently DVC
-supports Amazon S3, Microsoft Azure Blob Storage, Google Drive, Google Cloud
-Storage, SSH, HDFS, and other remote locations. The list is constantly growing.
-(For a complete list and configuration instructions, refer to `dvc remote add`.)
+DVC allows for a distributed environment and collaboration, making it easy to
+consistently relocate data files and directories into any machine, along with
+matching source code using Git. The key is to setup your own
+[remote storage](/doc/command-reference/remote), and push the data there so
```

Comment:
Step 0: I started rewriting the intro and every other section I combined into a single S3 remote example which we can decide whether to leave or remove later.